tools/data_manipulation/read_netcdf/read_netcdf/README.md

```diff
@@ -0,0 +1,23 @@
+# Netcdf reading and extraction
+## Netcdf info
+The first tool "netcdf info" use the C library netcdf (v4.5.0) to open and get general informations, variables names and attributes.
+Variables that can be extracted and dimensions availables are printed in tabular files.
+
+The tool use the bioconda tool netcdf-metadata-info : https://github.com/Alanamosse/Netcdf-Metadata-Info
+
+The tool also print a general information file. It's the result of the
+
+```
+ncdump -h inputfile
+```
+command.
+
+
+## Netcdf read
+The second tool "netcdf read" use the Python module Netcdf4 (1.3.1) which is an interface of the C library.
+
```

Comment:
I would suggest to use xarray instead. I think all the functions you mention are already implemented. In addition, with large files, xarray can chunk data with dask.

Reply:
sure.. just I am not a Python ar xarray guy ;) Do you think we can validate a first "old school" version as this, then you can propose to enhance it or this is better you directly modify it in this PR to have this incredible chance to have an amazing optimised tool ? ;)

Reply:
This make me think about the fact that it appears to me of interest @annefou to test adding Pangeo as a Jupyter notebook as Galaxy interactiveTool. Are you using it? Is it interesting for your communities ?

Reply:
Sure. I can rewrite it but I don't fully understance the xml part and what functions you really need.

Reply:
We have it already (https://live.usegalaxy.eu/?tool_id=interactive_tool_climate_notebook).

Reply:
but not pangeo related isn't it ?

Reply:
> 
> 
> Sure. I can rewrite it but I don't fully understance the xml part and what functions you really need.

hmm. ok, maybe we can leave this netCDF tool as is and think about a way to create future Galaxy tools based on xarray / dask no ? Here the idea is for an ecolog to have a possibility to sample pieces of netCDF info from another files with cordinates of sampling points...

Reply:
With the list of packages used in pangeo (except intake related; we haven't worked on pangeo-data yet)